crates/api_crud/src/comment/create.rs

```diff
@@ -143,8 +144,15 @@ impl PerformCrud for CreateComment {
     )
     .await?;
 
+    let person_id = local_user_view.person.id;
+    let comment_id = inserted_comment.id;
+    let comment_view = blocking(context.pool(), move |conn| {
+      CommentView::read(conn, comment_id, Some(person_id))
+    })
+    .await??;
+
     // If its a comment to yourself, mark it as read
-    if local_user_view.person.id == inserted_comment.creator_id {
```

Comment:
Not sure how we missed this. This means that every new comment is getting marked as read.